addons/packages/packetfence.spec

```diff
@@ -140,7 +140,7 @@ requires: perl(Const::Fast)
 # Perl core modules but still explicitly defined just in case distro's core perl get stripped
 Requires: perl(Time::HiRes)
 # Required for inline mode.
-Requires: ipset = 6.38, ipset-symlink
+Requires: ipset >= 7.1, ipset-symlink
```

Comment:
As discussed in the internal email, this is a bad idea because:

- It prevents us from releasing maintenance binaries for previous PF versions
- It needs to be pinned `= 7.1` if we decide to upgrade it so that an OS update will not break the binaries on the system